pkg/commands/pause.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Praetorian Security, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"github.com/praetorian-inc/trident/pkg/db"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"strconv"
+)
+
+var pauseCommand = &cobra.Command{
+	Use:   "pause",
+	Short: "pause campaign execution",
+	Long:  `can be used to temporarily pause a running campaign.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		pausePost(cmd, args)
+	},
+}
+
+func init() {
+	pauseCommand.Flags().StringVarP(&campaignID, "campaign", "c", "",
```

Comment:
The pflags library supports a few base types in addition to just strings to help with the strconv problem. We could change this to [IntVarP](https://godoc.org/github.com/spf13/pflag#IntVarP) to avoid the strconv in `pausePost`.

Reply:
I made this into a UintVarP since there's never a negative campaign and we treat the value as a uint in other parts of the code. If this is an issue let me know.